client/my-sites/checkout/composite-checkout/test/composite-checkout.js

```diff
@@ -790,7 +769,7 @@ describe( 'CompositeCheckout', () => {
 			screen
 				.getAllByLabelText( 'WordPress.com Personal' )
 				.map( ( element ) => expect( element ).toHaveTextContent( 'R$144' ) );
-			expect( screen.getAllByText( 'Domain Mapping: billed annually' ) ).toHaveLength( 2 );
+			expect( screen.getAllByText( 'Domain Mapping: billed annually' ) ).toHaveLength( 1 );
```

Comment:
Since the Order Review does not have active/inactive steps it is not rendering the inactive step with `display: none` anymore, so many counts need to be reduced in these tests.